docs/src/domains/steps/custom-non-production-domains.md

```diff
@@ -145,6 +150,9 @@ title=In the Console
    If the `Dev` environment is later removed,
    the `development.mysite.com` custom domain is removed too.
```

Comment:
`development.mysite.com` needs to be replaced by `mydev.com` to match with the example above.
Same for the other `development.mysite.com` 2 lines above.